.github/workflows/sycl-cts-on-spirv-be.yml

```diff
@@ -0,0 +1,77 @@
+# This workflow builds SYCL-CTS with -fsycl-use-spirv-backend-for-spirv-gen and
+# runs it with opencl:cpu & gen12 and level_zero:gpu & PVC on Sundays.
+
+name: SYCL-CTS with SPIR-V backend
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 7'
+
+permissions: read-all
+
+jobs:
+  ubuntu2204_build:
+    if: github.repository == 'intel/llvm'
+    uses: ./.github/workflows/sycl-linux-build.yml
+    secrets: inherit
+    with:
+      build_cache_root: "/__w/"
+      build_artifact_suffix: default
+      build_configure_extra_args: '--hip --cuda'
```

Comment:
do we need HIP and CUDA?

Reply:
Changed this to empty line to override default parameters - `--hip --cuda --native_cpu` - [99b8398](https://github.com/intel/llvm/pull/16731/commits/99b8398f719b2d9c8139843024942481c3b4b578)